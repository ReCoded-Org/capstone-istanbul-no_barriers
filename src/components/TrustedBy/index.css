:root {
  --trusted-background-color: #eee;
}

.trustedBy {
  background-color: var(--trusted-background-color);
  padding: 2rem;
}

.trustedBy img {
  height: 7.25rem;
}
