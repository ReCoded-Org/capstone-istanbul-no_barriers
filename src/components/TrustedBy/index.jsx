import React from "react";
import Container from "react-bootstrap/Container";
import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";
import { useTranslation } from "react-i18next";
import * as logo1 from "./932761.png";
import * as logo2 from "./932762.png";
import * as logo3 from "./932763.png";
import * as logo4 from "./932764.png";
import * as logo5 from "./932765.png";
import "./index.css";

const TrustedBy = () => {
  const { t } = useTranslation();
  return (
    <Container fluid className="trustedBy">
      <h4>{t("homePage.trustedBy")}</h4>
      <br />
      <Row>
        <Col>
          <img src={logo1} alt="logo1" title="logo1" />
        </Col>
        <Col>
          <img src={logo2} alt="logo2" title="logo2" />
        </Col>
        <Col>
          <img src={logo3} alt="logo3" title="logo3" />
        </Col>
        <Col>
          <img src={logo4} alt="logo4" title="logo4" />
        </Col>
        <Col>
          <img src={logo5} alt="logo5" title="logo5" />
        </Col>
      </Row>
    </Container>
  );
};

export default TrustedBy;
