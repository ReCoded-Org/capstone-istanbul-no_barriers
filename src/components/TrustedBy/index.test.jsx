import React from "react";
import renderer from "react-test-renderer";
import TrustedBy from "./index.jsx";

it("renders TrustedBy correctly", () => {
  const tree = renderer.create(<TrustedBy />).toJSON();
  expect(tree).toMatchSnapshot();
});
