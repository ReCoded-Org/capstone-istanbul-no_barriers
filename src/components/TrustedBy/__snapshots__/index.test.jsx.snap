// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TrustedBy correctly 1`] = `
<div
  className="trustedBy container-fluid"
>
  <h4>
    homePage.trustedBy
  </h4>
  <br />
  <div
    className="row"
  >
    <div
      className="col"
    >
      <img
        alt="logo1"
        src={
          Object {
            "default": "932761.png",
          }
        }
        title="logo1"
      />
    </div>
    <div
      className="col"
    >
      <img
        alt="logo2"
        src={
          Object {
            "default": "932762.png",
          }
        }
        title="logo2"
      />
    </div>
    <div
      className="col"
    >
      <img
        alt="logo3"
        src={
          Object {
            "default": "932763.png",
          }
        }
        title="logo3"
      />
    </div>
    <div
      className="col"
    >
      <img
        alt="logo4"
        src={
          Object {
            "default": "932764.png",
          }
        }
        title="logo4"
      />
    </div>
    <div
      className="col"
    >
      <img
        alt="logo5"
        src={
          Object {
            "default": "932765.png",
          }
        }
        title="logo5"
      />
    </div>
  </div>
</div>
`;
